webapp/components/pdf_preview.jsx

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import FileInfoPreview from './file_info_preview.jsx';
+
+import * as Utils from 'utils/utils.jsx';
+
+import loadingGif from 'images/load.gif';
+
+import React from 'react';
+import PDFJS from 'pdfjs-dist';
```

Comment:
I think imports would be the only exception, they were using `PDFJS` in all their documentation. 0/5